sql-project-1/sql-project-1.sql

```diff
@@ -0,0 +1,76 @@
+use dsstudent;
+
+create table customer_seraphima_ruiz (
+	customer_id smallint,
+	name varchar(20),
+	location varchar(20),
+	total_expenditure varchar(20)
+);
+
+insert into customer_seraphima_ruiz
+values
+(1701, "John", "Newport Beach, CA", 2000),
+(1707, "Tracy", "Irvine, CA", 1500),
+(1711, "Daniel", "Newport Beach, CA", 2500),
+(1703, "Ella", "Santa Ana, CA", 1800),
+(1708, "Mel", "Orange, CA",1700),
+(1716, "Steve", "Irvine, CA", 18000);
+
+update customer_seraphima_ruiz
+set total_expenditure = 1800
+where total_expenditure = 18000;

```

Comment:
Filter by primary key instead (could be multiple observations with total_exp=18000)